packages/pn-personafisica-webapp/src/components/Contacts/AddSpecialContactDialog.tsx

```diff
@@ -64,18 +69,37 @@ const AddSpecialContactDialog: React.FC<Props> = ({
   const getOptionLabel = (option: Party) => option.name || '';
   const [senderInputValue, setSenderInputValue] = useState('');
   const [alreadyExistsMessage, setAlreadyExistsMessage] = useState('');
-  const contactType = channelType.toLowerCase();
+  const [errorMessages, setErrorMessages] = useState<string | null>(null);
   const parties = useAppSelector((state: RootState) => state.contactsState.parties);
 
-  const checkIfSenderIsAlreadyAdded = (senders: Array<Party>) => {
-    const alreadyExists = digitalAddresses.some(
-      (addr) =>
-        addr.value !== value &&
-        addr.senders.some((sender) => senders.some((s) => s.id === sender.senderId))
-    );
+  const addressTypes: Array<AddressTypeItem> = allowedAddressTypes.map((addressType) => ({
+    id: addressType,
+    value: t(`special-contacts.${addressType.toLowerCase()}`, { ns: 'recapiti' }),
+    disabled: !digitalAddresses
```

Comment:
digitalAddresses is valued with the list of special addresses so the expression is always true. you can rename digitalAddresses with addressData and pass the result of ` const addressData = useAppSelector(contactsSelectors.selectAddresses);`. In this way, you can use `addressData.specialAddresses` where you need and, instead of to do this filter, you can check if `addressData[default_${addressType}Address]` exists